### PR TITLE
Yes_18_Send_MAP_based_on_other_order_statuses_Woocommerce

### PR DESCRIPTION
**Pull request by ijazalise**
done.

### CODE REVIEW

**Comment by aberenshtein** [comment]
@ijazalise please rebase

File: woocommerce-yotpo/wc_yotpo.php [hunk 1]
```diff
@@ -328,6 +328,37 @@ function wc_yotpo_past_order_time_query( $where = '' ) {
...
+                        }
+                    }
+                if ($is_success) {
+                    wc_yotpo_display_message('Past orders sent successfully', false);
+                    $yotpo_settings['show_submit_past_orders'] = false;
+                    update_option('yotpo_settings', $yotpo_settings);
+                }
+            }
+        } else {
+            wc_yotpo_display_message('Could not retrieve past orders', true);
```
Comment (ohadsc) [inline woocommerce-yotpo/wc_yotpo.php]: @sa3iedsabbah why is true in the second input param of wc_yotpo_display_message ?
seems like it's an error when retrieve past orders fails

File: woocommerce-yotpo/wc_yotpo.php [hunk 2]
```diff
@@ -388,17 +419,18 @@ function wc_yotpo_conversion_track($order_id) {
...
-				  'widget_location' => 'footer',
-				  'language_code' => 'en',
-				  'widget_tab_name' => 'Reviews',
-				  'bottom_line_enabled_product' => true,
-				  'bottom_line_enabled_category' => false,
-				  'yotpo_language_as_site' => true,
-				  'show_submit_past_orders' => true,
-				  'disable_native_review_system' => true,
-				  'native_star_ratings_enabled' => 'no');
+    return array('app_key' => '',
```
Comment (ohadsc) [inline woocommerce-yotpo/wc_yotpo.php]: @sa3iedsabbah can the indentation be on the same vertical line (including 'app_key')?

File: woocommerce-yotpo/wc_yotpo.php [hunk 3]
```diff
@@ -328,6 +328,37 @@ function wc_yotpo_past_order_time_query( $where = '' ) {
 }
 
 function wc_yotpo_send_past_orders() {
+    $yotpo_settings = get_option('yotpo_settings', wc_yotpo_get_degault_settings());
```
Comment (ohadsc) [inline woocommerce-yotpo/wc_yotpo.php]: a bit long function, is it possible to divide this function to 2-3 shorter functions? 
e.g. move block inside 'foreach ($past_orders as $post_bulk)' to separate function

**Comment by aberenshtein** [comment]
@ijazalise please review Ohad's code review comments

File: woocommerce-yotpo/wc_yotpo.php (hunk 1 above)
Comment (ijazalise) [reply woocommerce-yotpo/wc_yotpo.php]: i have no idea because its already there.

File: woocommerce-yotpo/wc_yotpo.php (hunk 3 above)
Comment (haroon-chaudhry) [reply woocommerce-yotpo/wc_yotpo.php]: This has been the legacy code, we didnt make many changes here. But if you want it to be refactored then please create a ticket for that.

File: woocommerce-yotpo/wc_yotpo.php (hunk 2 above)
Comment (haroon-chaudhry) [reply woocommerce-yotpo/wc_yotpo.php]: The indentation is correct if you open it in IDE, it must be the GitHub which is displaying it incorrectly.

**Comment by sa3iedsabbah** [comment]
Approved by QA

**Pull request merged**